packages/cardano-services/README.md

```diff
@@ -144,6 +144,15 @@ RABBITMQ_SRV_SERVICE_NAME=some-domain-for-rabbitmq \
 The _Docker images_ produced by the SDK and the _docker compose infrastructures_ (_mainnet_, _preprod_ and _local-network_) it includes are ready to be used in
 production environment.
 
+Running [cardano-node](https://github.com/input-output-hk/cardano-node/releases/tag/1.35.7) on _mainnet_ requires a minimum of 24GB RAM.
```

Comment:
```suggestion
Running [cardano-node] on _mainnet_ requires a minimum of 24GB RAM.
```

Best to not add maintenance overhead here. The suggestion will add the link sourced [here](https://github.com/input-output-hk/cardano-js-sdk/pull/827/files#diff-8190db84f584e67953bdc740053321b6a4d4ede419843a7bd47f90bab2729fc4R208)

Reply:
Nice, thank you!
5c4d690 should solve this